docs/learn/learn-nominator.md

```diff
@@ -211,13 +211,47 @@ more information and instructions for claiming rewards.
 
 ### What to Take Into Consideration When Nominating
 
-One thing to keep in mind as a nominator is the validator's commission. The commission is the
-percentage of the validator reward which is taken by the validator before the rewards are split
-among the nominators. As a nominator, you may think that the lowest commission is best. However,
-this is not always true. Validators must be able to run at break-even in order to sustainably
-continue operation. Independent validators that rely on the commission to cover their server costs
-help to keep the network decentralized. Commission is just one piece of the puzzle that you should
-consider when picking validators to nominate.
+There are many factors to consider when deciding which validator's to nominate. One useful tool for
+assisting in this process is the Staking [Targets](https://polkadot.js.org/apps/#/staking/targets)
+table. This displays potential validators in a table that can be evaluated and sorted using various
+metrics. Outlined below are the relevant columns to consider, followed by a brief description of
+each.
+
+- **payout**: How recently the validator has made it's last reward payout to nominators.
+- **nominators**: This column consists of two number values.
+
+  1) Amount of nominators currently bonded in the current era and considered `active`.
+
+  2) Total amount of nominators that nominated that validator or `all` validators.
+
+  - You may want to avoid validators with high subscriptions as you won't receive rewards if the
+    validator becomes oversubscribed, exceeding the maximum threshold of 256.
+  - Every nominator can select up to 16 validators but their bond is bonded to a single validator
+    each era. Therefore, not all nominators are actively bonded to a given validator during every
+    era. For additional information on the selection process checkout the research behind
+    [nominated proof-of-stake](https://research.web3.foundation/en/latest/polkadot/NPoS/1.%20Overview.html#polkadot-npos-1-overview--page-root).
```

Comment:
The reason a nominator's stake may not be applied to the validators in the active set is because they nominated too few validators (as the nominator is giving very little flexibility to the election algorithm to distribute their stake). A nominator can maximize the probability of having their stake applied to the validators in the active set by nominating 16 validators (the maximum allowed). 

Also, the election algorithm typically matches it with a single validator, but it can match with multiple validators too. Think of scenarios when someone with 3 Million DOT nominating  

- 1 validator - the election algorithm has no choice but to apply that stake in full
- 2 validators - the election algorithm can split the 3 Million DOT between these two validators based on other nominator account preferences. 2 million DOT to one validator with less nominators and 1 million DOT to a validator with more nominators etc.

Ultimately, the algorithm tries to maximize the overall network stake and minimizes the variance of active stake of each validator. If you check the stake on active validators today, it is 1.8 - 2.2 MDOT

Just rephrase the argument, as it is incorrect.

Reply:
Very interesting, thank you for the detailed explanation!